src/components/AddItems.jsx

```diff
@@ -60,14 +45,18 @@ export function AddItems() {
 					type="text"
 					id="item-name"
 					placeholder="Enter item name"
-				>
-					Item Name:
-				</TextInputElement>
+					label="Item Name:"
+					required={true}
+				/>
 
```

Comment:
TextInputElement takes a label prop instead of children prop, as well as required prop